packages/browser/Makefile

```diff
@@ -65,10 +64,6 @@ ci:
 	bash ./scripts/ci.sh
 .PHONY: ci
 
-size: ## Verify the final size of Analytics-Next
```

Comment:
Do we still do the size check somewhere?

Reply:
Yes, that make target wasn’t being used

Reply:
Ah, I’ve been using that, but can switch to yarn :smile: